src/pages/EventDetailsPage/components/EventDetails/styles.js

```diff
@@ -4,33 +4,24 @@ const styles = () => ({
     flexDirection: 'column',
     alignItems: 'center',
   },
-  container: {
-    backgroundColor: 'white',
+  eventDetailsCard: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    width: '536px',
```

Comment:
uh oh

Reply:
I'll see what I can do with that. For now I'll worry about all the logic and functionality stuff first, then I'll move on to making it look relatively nicer once I'm done with that